### PR TITLE
Issue 3964 fix - added line to about.js file

### PR DESCRIPTION
**Pull request by bfallonf**
As per #3964 

@jwforres can i get an ack that this is what you meant? And is this for OCP 3.4? The original issue reported it was found in Origin 1.4.

Then I'll merge. Thanks!

### CODE REVIEW

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -256,6 +256,7 @@ assetConfig:
   extensions:
     - name: about
       sourceDirectory: /path/to/about
+  hawtioPluginLoader.addModule('aboutPageExtension');
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: this needs to go above with the code for the angular module, this is content within the about.js file

**Comment by bfallonf** [comment]
@jwforres sure. Thanks. I've moved it above to the bottom of the file. Let me know if it's meant for a different position.

But also, this is for OCP 3.4?

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -227,6 +227,7 @@ angular
       .when('/about', {
         templateUrl: 'extensions/about/about.html',
         controller: 'AboutController'
+        hawtioPluginLoader.addModule('aboutPageExtension');
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: sorry wasnt clear, should be the very last line in the whole code block, after the last `);`

**Comment by jwforres** [comment]
yeah i believe OCP 3.4 is when we first documented this extension

File: install_config/web_console_customization.adoc (hunk 2 above)
Comment (bfallonf) [inline install_config/web_console_customization.adoc]: Oh I see. It's shown again in the file too. I should've been more observant. Sorry! Thanks!

**Comment by bfallonf** [comment]
Ok. Thanks @jwforres . I've updated it, and I'll merge.

**Pull request merged**